packages/core/content-manager/server/src/history/models/history-version.ts

```diff
@@ -0,0 +1,48 @@
+import type { Model } from '@strapi/database';
+import { HISTORY_VERSION_UID } from '../constants';
+
+const historyVersion: Model = {
+  uid: HISTORY_VERSION_UID,
+  tableName: 'strapi_history_versions',
+  singularName: 'history-version',
+  attributes: {
+    id: {
+      type: 'increments',
+    },
+    contentType: {
+      type: 'string',
+      column: { notNullable: true },
+    },
+    // documentId is a reserved attribute name
+    relatedDocumentId: {
```

Comment:
is it a reserved attribute name if adding the model directly?

Reply:
It's probably not anymore, I'll remove the comment 👍